lib/active_admin/reform/forms.rb

```diff
@@ -0,0 +1,37 @@
+require 'active_support/concern'
+require_relative 'resource_controller/data_access'
+
+#
+module ActiveAdmin
+  #
+  module Reform
+    # Reform integration helpers
+    module Forms
+      extend ActiveSupport::Concern
+
+      protected
+
+      # Wrap resource into Reform::Form if needed
+      # @param resource [ActiveRecord::Base]
+      # @return [ActiveRecord::Base, Reform::Form]
+      def apply_form(resource)
+        apply_form? ? form_class.new(resource) : resource
+      end
+
+      private
+
+      def apply_form?
+        case action_name
+        when 'new', 'edit', 'update', 'create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.